core/trie/trie_test.go

```diff
@@ -164,7 +164,7 @@ func TestPutZero(t *testing.T) {
 		var keys []*felt.Felt
 
 		// put random 64 keys and record roots
-		for range 64 {
+		for range make([]int, 64) {
```

Comment:
Is this change required?

Reply:
I added this change because `golangci-lint` raise `cannot range over 64 (untyped int constant) (typecheck)`.
should i revert this part?

Reply:
reverted to previous state